src/platform/code-editor/code-editor.component.ts

```diff
@@ -626,7 +634,6 @@ export class TdCodeEditorComponent implements OnInit, AfterViewInit, ControlValu
         // append the webview to the DOM
         this._editorContainer.nativeElement.appendChild(this._webview);
     }
-    this._componentInitialized = true;
```

Comment:
Was set to true before the electron component emitted the `onEditorInitialized` event so moved where this is set to true.